src/north/console/Console.class.js

```diff
@@ -3,50 +3,52 @@ const fs = require('fs')
 const ApiHandler = require('../ApiHandler.class')
 
 class Console extends ApiHandler {
-  /**
+    static category = 'Debug'
```

Comment:
There seem to be an indentation problem here, please fix that (it should fix the rest of the file too)